README.md

```diff
@@ -23,6 +23,27 @@ and the Collector's [troubleshooting guide].
 [OpenTelemetry documentation]: https://opentelemetry.io/docs/collector/troubleshooting/
 [troubleshooting guide]: https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/troubleshooting.md
 
+## OpenTelemetry Sampling Considerations
```

Comment:
I think we could put that below the Component list since that one is limited in size and probably of higher interest for those looking into the distro the first time. This section here with best practices and troubleshooting tips might still grow a lot and could then expand further down.